sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.service.ts

```diff
@@ -119,6 +127,128 @@ export class SIMSToSFASService {
     return modifiedStudentIds;
   }
 
+  /**
+   * Get all student ids of students who have one or more updates in application related data.
+   * @param modifiedSince the date after which the student data was updated.
+   * @param modifiedUntil the date until which the student data was updated.
+   */
+  async getAllStudentsWithApplicationUpdates(
+    modifiedSince: Date,
+    modifiedUntil: Date,
+  ): Promise<ApplicationRecord[]> {
+    return (
+      this.applicationRepo
+        .createQueryBuilder("application")
+        .select("student.id", "studentId")
+        .addSelect("application.id", "applicationId")
+        .addSelect("programYear.programYear", "programYear")
+        .addSelect("offering.offeringIntensity", "offeringIntensity")
```

Comment:
Same question as we discussed before from business perspective, is the expectation of FT and PT application records to come in any order and (i.e FT PT records mixed without an order) or FT records followed by PT records or vice-versa?

Reply:
I clarified this with @CarlyCotton and there is no requirement to expect in particular order, as the record type will show the difference.